diff --git a/Dockerfile b/Dockerfile
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM nimlang/nim:2.2.6-alpine-regular as nim
+FROM ghcr.io/tiekoetter/nitter-nim:2.2.6-alpine-regular as nim
 LABEL maintainer="setenforce@protonmail.com"
 
 RUN apk --no-cache add libsass-dev pcre
@@ -13,7 +13,7 @@ RUN nimble build -d:danger -d:lto -d:strip --mm:refc \
     && nimble scss \
     && nimble md
 
-FROM alpine:latest
+FROM ghcr.io/tiekoetter/nitter-alpine:3.24.2
 WORKDIR /src/
 RUN apk --no-cache add pcre ca-certificates openssl
 COPY --from=nim /src/nitter/nitter ./
